kits/debugpilot-ai/README.md

```
@@ -0,0 +1,302 @@
+* Authentication token failures
+* Deployment/build pipeline failures
+* Database connection timeouts
+* Runtime exceptions
+* TypeScript build errors
+* Infrastructure-related logs
+
+---
+
+## 🖼️ Screenshots
+
+### Landing Interface
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)